ang/test/mocks/services/formatActivity.mock.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env jasmine */
+
+(function () {
+  var module = angular.module('civicase');
+
+  module.factory('formatActivity', function () {
+    return jasmine.createSpy('formatActivity')
+      .and.callFake(function (activity) {
+        activity.category = [];
```

Comment:
`category` is the only property that this mocked function needs to add to the whatever pretends to be an activity object, otherwise some tests will fail

I've originally mocked `formatActivity` and `formatCase` here https://github.com/compucorp/uk.co.compucorp.civicase/blob/3cff1250dccd5bddf4e4f8bea32ee3f0efed9e75/ang/test/civicase/DashboardTab.spec.js#L624 (#90) and given that I needed to mock them again for the unit tests of the calendar, I thought it would be useful to have them always mocked (like it happens for `crmApi`, for example). I made sure that by mocking them all the tests would still pass